example/chatRoom.tsx

```diff
@@ -56,7 +52,7 @@ export const Chat = () => {
         />
       </div>
       <hr />
-      {chatRoomFish && (
+      {
```

Comment:
in some cases, it was very handy to have no fish in the beginning. 

"select a PO first" 
(without wrapping it into sub-components)

but this can come back with the split props